database/spamchannel.py

```diff
@@ -78,3 +78,93 @@ def dump(self) -> Dict[str, Union[int, str]]:
             "channel_id": self.channel_id,
             "primary": self.primary,
         }
+
+
+class SpamLimit(database.base):
+    __tablename__ = "spamlimits"
+
+    idx = Column(Integer, primary_key=True, autoincrement=True)
+    guild_id = Column(BigInteger)
+    channel_id = Column(BigInteger)
+    limit = Column(Integer)
```

Comment:
I'm not sure if we need this to be configurable. Maybe we should just set the best values and enforce them.

Reply:
I think thi is the way, 'cause you can enforce the rules differently (set easy limit in offtopic channels or set hard limits in channels where you don't want those commands at all.